app/src/main/java/com/infomaniak/drive/ui/MainViewModel.kt

```diff
@@ -440,4 +469,11 @@ class MainViewModel(appContext: Application) : AndroidViewModel(appContext) {
         realm.close()
         super.onCleared()
     }
+
+    data class FileRequest(
```

Comment:
Because in some cases, it's not a real api call we're doing. `FileResponse` suppose we have a response from a WS. Both of them are fine to me so if you prefer FileResponse, I'll change.

Reply:
FileRequest sounds too much like a request api, whereas here you need it as a response.
I don't know if we can find better, but in itself `FileResponse` for me it simply means the result of the file, wait maybe we can call it `FileResult`, I find that it works better